docs/manual/src/interface/namespace.md

```diff
@@ -0,0 +1,19 @@
+# Namespace
+
+Every UniFFI component has an associated *namespace* which is taken from
+the name of the inline module defining the interface:
+
+```rust
+#[uniffi::declare_interface]
+mod example_namespace {
+  // The resulting interface definition will have a namespace
+  // of "example_namespace".
+}
+```
+
+The namespace servces multiple purposes:
+- It defines the default package/module/etc namespace in generated foreign-language bindings.
+- It is used to ensure uniqueness of names in the generated C FFI.
```

Comment:
I initially thought "what does this mean?" then I saw the following sentence - consider combining that sentence into this one?